### DISCUSSION TITLE
Transmitter dropped

### DISCUSSION THREAD

**Discussion by gazhay** [NightscoutFoundation/xDrip#3466] [Q&A]
Been using xdrip with my nothing 2 phone & xdrip for months (and before that with pixel phone)

Last night i lost transmitter signal, nothing out of the ordinary from time to time.

Woke up 9 hours later and been no readings since.

Seems bluetooth link was broken.
Couldn't get anything to reconnect. Tried collector restart but eventually changed the HW id of the transmitter and it asked to pair again.

Battery B is reading low after 63 days, and the repair seems to have ended my sensor 5 days early.

Q1. How should I have re-paired?
Q2. Sensor won't start again, assume its dead?
Q3. With battery B in the red (233) and issues is it even worth trying with a new sensor or should i just get a new transmitter?

Thanks

Using dexcom one (g6)
Nothing 2 phone
![Screenshot_20240516-080354.png](https://github.com/NightscoutFoundation/xDrip/assets/871127/1bc85561-ebb9-4480-80cc-1c1cffaa2fe9)

**Comment by Navid200** [comment]
Follow these instructions to see how Voltage B has been changing over the past 9 days:
https://navid200.github.io/xDrip/docs/Dexcom/BatteryVoltageLogs.html

**Reply by gazhay** [reply]
![Screenshot_20240516-171750.png](https://github.com/NightscoutFoundation/xDrip/assets/871127/8722e0c6-ac2b-47cb-9a66-554d4f346279)

Seems like it started discharging somewhat quickly in last couple of days.

Think i won't risk a new sensor with it and just pick up a new transmitter tomorrow

**Reply by Navid200** [reply]
Yes, the battery is dying.
It's best to switch to a new transmitter.

If you get a transmitter, but don't use it, the battery still ages.
It's not a good idea to get a backup transmitter unless you are willing to put up with the possibility that you will not be able to use each for 110 days.

**Reply by gazhay** [reply]
I've never been able to get 110 days. They only last 90 anyway.
I think its better to have a backup near the EoL of the one being used to avoid my situation of days without sensor ymmv.

**Reply by Navid200** [reply]
That is incorrect.  They last 110 days.
It's the warranty that is for 90 days only.
Of course, it could last even less than 90 days if you get one and put it on a shelf as explained already.

I used G6 for 4 years before switching to G7.  Every single G6 I got in that period worked 110 days for me.
Also, if you don't know how to time your sessions, you may not be able to get the last session to end exactly on day 110.  In that case, it will work at least 100 days.

**Reply by gazhay** [reply]
I wasn't trying to start a fight.
I have only ever been told they last 90 days, i've never been motivated to challenge that as the prescription is always filled in time.
I am fairly sure xdrip issues warning about starting sensors > 80 days on the transmitter, which could also be cause for my never having tried to extend them.

I guess if you are out-of-pocket for the transmitters it makes sense you may try to exceed manufacturers spec.

As for warranty, dexcom are not interested in honouring warranty if you don't use their app anyway, so thats moot.

**Reply by Navid200** [reply]
I'm not interested in a fight.
Incorrect information about xDrip or devices will be corrected in these forums because many can find the posts through search.
This is a forum.  So, you can disagree as long as you keep it civil.  

These forums were opened so that we can help xDrip users.
I asked for these forums to be opened for that reason.  
I am here to make sure it does its job.

You don't have to accept what I have stated here.

You brought up the figure of 90 days.  I stated the only significance of that figure was how long the warranty was valid.
I never meant whether Dexcom will honor it or not.  

Yes, the color of the number of days shown in xDrip changes as the number increases.  I made that change to xDrip as a warning to remind us to plan for ordering the next transmitter.
The color also changes to red when it reaches 100, which is when you cannot start a sensor any longer, hence the limit of 110 days if you start the last one when it shows 99.

You are more than welcome to do whatever you wish.  No one is trying to convince you to do otherwise.
However, facts are facts.  
The statement "a G6 transmitter only lasts 90" is not a fact, and will be corrected.


You don't have to agree with me.
If you can show me how you have established that transmitters only last 90 days, I will be grateful.

This shows my measurement results of some of my G6 transmitter battery voltages even after they stopped working:
https://navid200.github.io/xDrip/docs/Battery-condition.html

This page contains different topics about Dexcom and how to use a Dexcom through xDrip:  https://navid200.github.io/xDrip/docs/Dexcom_page.html
I will be more than happy to make corrections when needed.  People point out errors and I correct them if they are wrong.  Otherwise, we talk about it without making it personal.  The enemy is diabetes, not you nor me.  I don't have a fight with you and I don't want to have a fight with you.

**Reply by Navid200** [reply]
This page presents my knowledge of G6 lifetime:
https://navid200.github.io/xDrip/docs/Transmitter-lifetime.html
It also shows the status page when I was on last day at day 109.  Considering during the first day, it shows 0, 109 means 110 days.

**Reply by Navid200** [reply]
I am happy to remove all my previous comments if you remove yours where you state they last 90 days.
It's your choice.

**Reply by gazhay** [reply]
You seem fully capable of deleting my responses so I don't think you need me to do anything.

Again, you are actively attacking a user who simply stated that the dexcom lasted for 90 days in line with my only knowledge at the time - that being from dexcom the manufacturer and all the healthcare workers who have educated me on the transmitter.

I wasn't claiming they *couldn't* last longer (otherwise why would I have bothered to day I've never gotten 110 days).

Remove what you like, including your overreaction to thinking i was distributing some kind of authoritative fake news.

Please also consider reviewing your actions. It demeans open source to have such interactions based on what is clearly misunderstanding.

**Reply by Navid200** [reply]
That's OK.  Misunderstandings happen.  I'm sorry about my response offending you.
Please come back here if you need help with xDrip and we will do what we can to help.
